bundles/fluent-en_GB/atrium-fluent-en_GB/build.gradle.kts

```diff
@@ -16,11 +21,17 @@ kotlin {
                 api(prefixedProject("translations-en_GB"))
             }
         }
+        val commonTest by getting {
+            dependencies {
+                implementation("io.kotest:kotest-runner-junit5:$kotestVersion") // for kotest framework
+                implementation("io.kotest:kotest-property:$kotestVersion") // for kotest property test
```

Comment:
remove this line for now, we don't have any property tests in Atrium so far. We can add it again once we use it